### PR TITLE
Improved token loading, removed card price code

### LINKED ISSUE #472: Loading from clipboard puts tokens in Maindeck
As title, creating any deck from a clipboard list won't put tokens that match the user's token database into the tokens section of their deck.

This appears to be the only way to put tokens anywhere but in the 'tokens' section, but surely there's a better way to support the [Horde format](http://mtgsalvation.gamepedia.com/Horde_Magic)?

### LINKED ISSUE #1599: Saving decks in text format puts tokens in main deck
So the last few times ive put tokens in the sideboard for a deck, when i load the deck in a game, for some reason the token keep appeaping in the main deck??

### PR DESCRIPTION
**Pull request by ctrlaltca**
## Related Ticket(s)
- Fixes #472
- Fixes #1599

## Short roundup of the initial problem
When loading a "plain text" deck from a file or clipboard, tokens were not recognized and added to the main deck

## What will change with this Pull Request?
Tokens will be correctly added to the "tokens" deck zone.
Additionally, there were a lot of zone codes ("main", "side", "tokens") scattered around the code. They have been replaced by three new constants `DECK_ZONE_*`.

### CODE REVIEW

**Comment by ZeldaZach** [comment]
I pushed an additional change to this PR. It removes all traces of card pricing as we haven't used this feature for a long time and it's just scrap and this code was currently doing some stuff with it that was unneeded.

**Comment by ctrlaltca** [comment]
I don't get why you added unrelated changes to this PR, instead of creating a new one.

**Comment by ZeldaZach** [comment]
I saw you edited some parts of the code that dealt with pricing and I figured it was time to remove it, and I didn't want a merge conflict to arise. I can revert if you'd like and push your changes in and make a new PR later

**Comment by ctrlaltca** [comment]
Don't worry, that's fine

**Comment by ZeldaZach** [comment]
@Daenyth Can you get around to this at some point this weekend?

File: cockatrice/CMakeLists.txt [hunk 1]
```diff
@@ -97,9 +97,8 @@ SET(cockatrice_SOURCES
     src/thememanager.cpp 
     src/localserver.cpp 
     src/localserverinterface.cpp 
-    src/localclient.cpp 
-    src/priceupdater.cpp 
-    src/qt-json/json.cpp
+    src/localclient.cpp
+        src/qt-json/json.cpp
```
Comment (Daenyth) [inline cockatrice/CMakeLists.txt]: indent

**Comment by ZeldaZach** [comment]
@Daenyth Done, good to go. I'll merge and not squash when it's ready or you can do it now.

**Pull request merged**